modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/GmosControllerEpics.scala

```diff
@@ -44,10 +44,10 @@ object GmosControllerEpics extends GmosSouthController {
 
   implicit val ampGainSettingEncoder: EncodeEpicsValue[AmpGainSetting, String] = EncodeEpicsValue(_.value.toString)
 
-  implicit val ampCountEncoder: EncodeEpicsValue[AmpCount, Int] = EncodeEpicsValue {
-    case AmpCount.THREE  => 3
-    case AmpCount.SIX    => 6
-    case AmpCount.TWELVE => 12
+  implicit val ampCountEncoder: EncodeEpicsValue[AmpCount, String] = EncodeEpicsValue {
+    case AmpCount.THREE  => "BEST"
+    case AmpCount.SIX    => "ALL"
+    case AmpCount.TWELVE => "SECONDARY"
```

Comment:
Yes, I just worked on this, and THREE is an option in the OCS code, but only for, IIRC, E2V detectors. I think now that we are using Hamamatsu CCDs for both GMOS-N and GMOS-S, THREE may be obsolete, but I am not certain.